examples/experimental/i3s-17-and-debug/mesh-layer/mesh-layer.js

```diff
@@ -99,6 +96,7 @@ export default class MeshLayer extends SimpleMeshLayer {
   }
 
   getModel(mesh) {
+    const {segmentationMode, segmentationData} = this.props;
```

Comment:
Not sure that calling the prop `segmentationMode` is the most intuitive name... How about `pickFeatures`?

Reply:
renamed